src/utilities/theme/context.tsx

```diff
@@ -1,4 +1,4 @@
 import React from 'react';
-import {Theme} from './types';
+import {Logo} from './types';
 
-export const ThemeContext = React.createContext<Theme | undefined>(undefined);
+export const ThemeContext = React.createContext<Logo | undefined>(undefined);
```

Comment:
This doesn't seem right - My initial thoughts would have expected the values in the context to include all the colors so you can do things like `useContext(ThemeContext)['colors']['surface']`. Is that not desired?

The naming of the stuff within themeing as always been a bit obscure to me - I've never been very clear on what is publicly exposed vs what is internal. Might be good to have a 30min chat so I can understand that better

Reply:
I went ahead and reverted this back to `Theme`, and stopped destructuring colors off the theme we provide through context.